packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx

```diff
@@ -6,11 +6,15 @@ import {
   ContextualMenu,
   DirectionalHint,
   IContextualMenuItem,
+  IPersona,
+  Persona,
+  PersonaSize,
   Target,
   useTheme
 } from '@fluentui/react';
 import { _formatString } from '@internal/acs-ui-common';
 import React, { useMemo } from 'react';
+import { OnRenderAvatarCallback } from '../..';
```

Comment:
avoid importing from the `src` or `components` index.ts files, instead use a more complete path here:
`../../types`

Reply:
(it can cause circular dependency issues, particularly in unit tests)

Reply:
updated